src/consensus/mod.rs

```diff
@@ -18,4 +18,7 @@
 //! conform to Bitcoin consensus.
 //!
 
+pub mod encode;
 pub mod params;
+
+pub use self::encode::{Encodable, Decodable};
```

Comment:
Should also `pub use` Encoder and Decoder (suggested renames of SimpleEncoder and SimpleDecoder)

Reply:
and `deserialize` and `serialize`

Reply:
Addressed in: cccbb057bc88462284430b91ea333ea13ecbf7b0